install.sh

```diff
@@ -57,33 +83,42 @@ chroot_install() {
         || exit 1
     chmod +x ./alpine-chroot-install || exit 1
     ./alpine-chroot-install -d /alpine-v310 -b v3.10
-    /alpine-v310/enter-chroot install.sh
+    /alpine-v310/enter-chroot ./install.sh
+}
+
+check_os() {
+    if [ -f "/etc/alpine-release" ]; then
+        alpine_install || exit 1
+    elif [ -x "$(command -v apt-get)" ]; then
```

Comment:
I'd prefer:

```shell
if [ -f "/etc/alpine-release" ]; then
	alpine_install
	return 0
fi

if command -v apt-get >/dev/null; then
	debian_install
	return 0
fi

echo "Not an Alpine or Debian-based Linux system"
echo
echo "Attempting to create Alpine chroot"
chroot_install
```

Reply:
Agreed, these separate if's are less coupled and probably easier to maintain